Irina/src/main/java/oop/CompanyOrder.java

```diff
@@ -0,0 +1,10 @@
+package oop;
+
+public class CompanyOrder extends OrderByAnyUser {
+
+    private Company company;
```

Comment:
How would you initiate this field? I don't see any setters or constructors here

Reply:
Done